Sources/Cache/CacheSerializer.swift

```diff
@@ -80,8 +81,22 @@ public struct DefaultCacheSerializer: CacheSerializer {
     
     /// The default general cache serializer used across Kingfisher's cache.
     public static let `default` = DefaultCacheSerializer()
-    private init() {}
-    
+
+    /// The compression quality when converting image to a lossy format data. Default is 1.0.
+    public var compressionQuality: CGFloat = 1.0
```

Comment:
Does it have to be of `CGFloat`? Can't it be a `Double` as it was?

Reply:
compressionQuality might be defined as an enumeration with a default value set to high.
`public var compressionQuality: JPEGCompressionLevel = .high`
```
enum JPEGCompressionLevel {
  case custom(Double)
  case max, high, med, low

  var rawValue: Double {
    switch self {
    case .max:
      return 1.0
    case .high:
      return 0.9
    case .med:
      return 0.5
    case .low:
      return 0.2
    case .custom(let customValue):
      return customValue
    }
  }
}
```

Reply:
`UIImage` receives a `CGFloat` when converting to JPEG data: https://developer.apple.com/documentation/uikit/uiimage/1624115-jpegdata

So I guess keeping them aligned would reduce the mental pressure of users.

What do you mean by "a `Double` as it was"? I don't think this value once being a `Double` anywhere... 😕

Reply:
I have made the mistaken assumption about the original type of the compressionQuality property because CoreGraphics library was imported.
Below, updated code proposition.
```
enum JPEGCompressionLevel {
  case custom(CGFloat)
  case max, high, med, low

  var rawValue: CGFloat {
    switch self {
    case .max:
      return 1.0
    case .high:
      return 0.9
    case .med:
      return 0.5
    case .low:
      return 0.2
    case .custom(let customValue):
      return customValue
    }
  }
}
```

Example of use:
var cacheSerializer = DefaultCacheSerializer()
cacheSerializer.compressionQuality = .max
cacheSerializer.compressionQuality = .custom(0.8)